flowzone.yml

```diff
@@ -1032,12 +1092,13 @@ jobs:
         with:
           node-version: 18.x
 
-      - name: Install versionist
+      - name: Install versioning tools
         if: inputs.disable_versioning != true
         run: |
-          npm install -g balena-versionist@~0.14.13 versionist@^7.0.3
-          npm ls -g balena-versionist
-          npm ls -g versionist
+          cat <package.json | jq -r '.dependencies | to_entries[] | .key + "@" + .value' \
```

Comment:
This would be looking at the `package.json` of the repo we're building, *not* of flowzone, so this won't work as expected outside of the flowzone repo